File: JavaScript/Practice/LeetCode/Easy/Sum of Digits in Base K.js
// https://leetcode.com/problems/sum-of-digits-in-base-k/description/

/**
 * @param {number} n
 * @param {number} k
 * @return {number}
 */
var sumBase = function(n, k) {
    let sum = 0
    while(n>0){
        sum += n%k
        n = ~~(n/k)
    }
    return sum    
};

File: C#/Practice/LeetCode/Easy/Sum of Digits in Base K.cs
// https://leetcode.com/problems/sum-of-digits-in-base-k/description/

public class Solution {
    public int SumBase(int n, int k) {
        int sum = 0;

        while ( n > 0){
            sum += n%k;
            n = ~~(n/k);
        }

        return sum;       
    }
}